app/react/components/Widget/Widget.tsx

```diff
@@ -1,20 +1,29 @@
-import { createContext, PropsWithChildren, useContext } from 'react';
+import clsx from 'clsx';
+import { createContext, PropsWithChildren, Ref, useContext } from 'react';
 
 const Context = createContext<null | boolean>(null);
 Context.displayName = 'WidgetContext';
-
 export function useWidgetContext() {
   const context = useContext(Context);
-
   if (context == null) {
     throw new Error('Should be inside a Widget component');
   }
 }
-
-export function Widget({ children }: PropsWithChildren<unknown>) {
+export function Widget({
+  children,
+  className,
```

Comment:
Does `className` need to be added in this PR? It doesn't seem to be used as a prop

Reply:
I made both EE and CE component looks he same but I reverted it and just kept the ID